include/ofi_util.h

```diff
@@ -205,6 +205,7 @@ struct util_domain {
 	struct ofi_mr_map	mr_map;
 	enum fi_threading	threading;
 	enum fi_progress	data_progress;
+	struct ofi_ops_flow_ctrl *flow_ctrl_ops;
```

Comment:
Why was this moved from rxm to the general code?

Reply:
oops. yeah i'll move that back to into rxm. this was an oversight on my part.